src/CONST.ts

```diff
@@ -476,6 +476,7 @@ const CONST = {
         ACTIONS: {
             LIMIT: 50,
             TYPE: {
+                SUBMITTED: 'SUBMITTED',
```

Comment:
I know this is not alphabetical strictly but can you move it to the bottom based on alphabet :D 

Or even sort the list alphabetically now

Reply:
Yeah, I like the idea. Done - 8e3aabe643356dd493604d727e5699e93f323dac.